docs/source/api_index.rst

```diff
@@ -71,13 +71,13 @@ Analysis
     cart2pol
     pol2cart
 
-Move Accessor
--------------
+MovementDataset
```

Comment:
Something is off with the auto api, as it doesn't show all methods

Reply:
Issue opened here: https://github.com/neuroinformatics-unit/movement/issues/178